flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/plan/stream/table/ValuesTest.scala

```diff
@@ -0,0 +1,104 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.planner.plan.stream.table
+
+import org.apache.flink.table.api.DataTypes
+import org.apache.flink.table.api.scala._
+import org.apache.flink.table.planner.utils.TableTestBase
+import org.apache.flink.types.Row
+
+import org.junit.Test
+
+import java.util.{Arrays => JArrays}
+
+import scala.collection.JavaConverters._
+
+class ValuesTest extends TableTestBase{
+
+  @Test
+  def testValuesFromRowExpression(): Unit = {
+    val util = streamTestUtil()
+    val t = util.tableEnv.fromValues(
+      row(lit(1) + 3, "ABC"),
+      row(pi(), "ABC"),
+      row(3.1f, "DEF"),
+      row(99L, "DEFG"),
+      row(0d, "D")
+    )
+    util.verifyPlan(t)
+  }
+
+  @Test
+  def testValuesFromRowObject(): Unit = {
+    val util = streamTestUtil()
+    val t = util.tableEnv.fromValues(
+      Row.of(Int.box(1), "ABC"),
+      Row.of(Double.box(Math.PI), "ABC"),
+      Row.of(Float.box(3.1f), "DEF"),
+      Row.of(Long.box(99L), "DEFG"),
+      Row.of(Double.box(0d), "D")
+    )
+    util.verifyPlan(t)
```

Comment:
Can we also add one or two more complex ITCases? Esp. the casting of rows might not be supported. Just to make sure the end-to-end story works.